core/src/main/kotlin/io/github/serpro69/kfaker/provider/RandomProvider.kt

```diff
@@ -140,12 +145,23 @@ class RandomProviderConfig @PublishedApi internal constructor() {
     @PublishedApi
     internal val predefinedGenerators = mutableMapOf<KClass<*>, () -> Any>()
 
+    @PublishedApi
+    internal val nullableGenerators = mutableMapOf<KClass<*>, () -> Any?>()
+
     /**
      * Configures generation for a specific type. It can override internal generators (for primitives, for example)
      */
     inline fun <reified K : Any> typeGenerator(noinline generator: () -> K) {
         predefinedGenerators[K::class] = generator
     }
+
+    /**
+     * Configures generation for a specific nullable type. It can override internal generators (for primitives, for example)
+     */
+    @JvmName("typeGenerator_null")
+    inline fun <reified K : Any?> typeGenerator(noinline generator: () -> K?) {
```

Comment:
Maybe we could stick to java naming conventions and avoid `_` here in the `@JvmName` annotation.
I'm also thinking if having explicit "null" in the function name would be better as it closer follows Kotlin's conventions for naming functions that deal with nullables. So maybe `nullableTypeGenerator` would be better?

Reply:
done 👍